common/app/common/TrailsToShowcase.scala

```diff
@@ -105,24 +109,84 @@ object TrailsToShowcase {
   }
 
   def asSingleStoryPanel(content: PressedContent): Either[Seq[String], SingleStoryPanel] = {
-    val proposedTitle = titleOfLengthFrom(MaxLengthForSinglePanelTitle, content)
+    val proposedTitle = titleOfLengthFrom(MaxSinglePanelTitleLength, content)
     val proposedPanelTitle = panelTitleFrom(content)
     val proposedWebUrl = webUrl(content).map(Right(_)).getOrElse(Left(Seq("Trail had no web url")))
     val proposedImageUrl = singleStoryImageUrlFor(content)
-    val proposedBulletList =
-      content.card.trailText.map(extractBulletsFrom).getOrElse(Left(Seq("No trail text available")))
 
-    // Collect all mandatory values; any missing will result in a None entry
-    val proposedOverline = overlineFrom(content)
+    // Decide which type of panel to produce
+    // If supporting content is present try to map into into a related articles article group for a Related Articles panel
+    // If then try to produce a bullet item list for a Bullets panel
+    // This might get more interesting if we implement Key Moments but the principal is the same.
+    val supportingContent = content match {
```

Comment:
This is the gist of it... the presence of Sublinks triggers an alternative ending.

Reply:
👍